.storybook/stories/dropdown/dropdown-menu.stories.ts

```diff
@@ -70,6 +70,7 @@ const DropdownMenuTemplate: StoryFn = args => ({
   props: args,
 });
 
+/*
 const DropdownMenuAllTemplate: StoryFn = args => ({
```

Comment:
Why is this story commented out? Was this a mistake?

Reply:
Playwright was causing intermittent issues with full page screenshots for this story, reverted the change.